acrobat/styles/styles.css

```diff
@@ -23,6 +23,20 @@ div[data-section="widget"] {
   min-height: 570px;
 }
 
+#dc-converter-widget[verb=chat-pdf] {
+  background-color: #fff;
+  background-image: url(../img/backgrounds/chat-pdf.png);
```

Comment:
I suggested to the designers that we use a gradient instead. not sure whether loading a background image would affect our performance, but we can tweak this later